doc/DeveloperGuide.md

```diff
@@ -94,6 +94,11 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should work on both 32-bit and 64-bit environments
+6. Should have simple commands that user can understand.
```

Comment:
Here can be more specific, simple in what sense? Length? Language / Grammar? Etc.